unison-share-api/src/Unison/Server/Orphans.hs

```diff
@@ -65,11 +65,33 @@ instance FromJSONKey ShortHash where
         Nothing -> fail $ "Invalid Shorthash" <> Text.unpack txt
         Just sh -> pure sh
 
-deriving instance ToSchema ShortHash
-
 instance FromHttpApiData ShortBranchHash where
   parseUrlPiece = maybe (Left "Invalid ShortBranchHash") Right . SBH.fromText
 
+-- | Always renders to the form: #abcdef
+instance ToHttpApiData ShortHash where
+  toQueryParam = SH.toText
+
+-- | Accepts shorthashes of any of the following forms:
+-- @abcdef
+-- @@builtin
+-- #abcdef
+-- ##builtin
+-- abcdef
```

Comment:
So if we support all of these, then this is backwards compatible? If thats true we don't need to coordinate deploys.